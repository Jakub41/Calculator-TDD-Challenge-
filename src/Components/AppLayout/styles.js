import styled from "@emotion/styled";

export const ContentLayout = styled.div`
	width: 100px;
	height: 100px;
`
