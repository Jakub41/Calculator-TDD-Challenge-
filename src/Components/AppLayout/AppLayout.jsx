import React from "react";
import { ContentLayout } from "./styles";
import Calculator from "../Calculator";

const AppLayout = () => {
	return (
		<>
			<ContentLayout>
				<Calculator />
			</ContentLayout>
		</>
	);
};

export default AppLayout;
