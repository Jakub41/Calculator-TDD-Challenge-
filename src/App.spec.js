
import React from "react";
import renderer from "react-test-renderer";
import AppLayout from "./Components/AppLayout";

describe("App", () => {
	it("AppLayout Snapshot renderer", () => {
		const component = renderer.create(<AppLayout />);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
