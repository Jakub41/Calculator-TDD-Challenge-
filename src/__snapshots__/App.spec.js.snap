// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App AppLayout Snapshot renderer 1`] = `
<div
  className="css-14fwp1f"
>
  <div
    className="css-1esthja"
  >
    <div
      className="css-1esthja"
    >
      0
    </div>
    <div
      className="css-1esthja"
    >
      <div
        className="css-1esthja"
      />
      <div
        className="css-1esthja"
      />
    </div>
  </div>
</div>
`;
